tests/packed-load-64-2.ispc

```diff
@@ -0,0 +1,15 @@
+export void f_f(uniform float RET[], uniform float aFOO[]) {
+    uniform int64 a[programCount];
+    a[programIndex] = aFOO[programIndex];
+    int64 aa = 15;
+    uniform int count = 0;
+    if (programIndex < 2)
+        count += packed_load_active(a, &aa);
```

Comment:
`count` is dead code and is never checked.

Reply:
I basically used the same tests as i32. So, I have removed count for both. Existing test and the new one.

Reply:
Yeah. I removed it for this and the original int32 test from which this test was taken from.

Reply:
No, checking that return value is correct is important, otherwise we may easily miss the problem in popcnt() implementation. I think both tests need it.

Reply:
I think I idea behind these tests originally was each test tests something. For eg. 'packed-load-64-3.ispc' checks the popcnt calculation. Same with 'packed-load-64-5.ispc ' - it also kind of tests the popcnt in it's own way. 
So even though not all tests check the popcnt value, so do

Reply:
Ok. It's always good to spell out the intention of the test in the comments. Keep that in mind for new tests.